sdk/recoveryservices-backup/Microsoft.Azure.Management.RecoveryServices.Backup/src/Generated/BackupEnginesOperations.cs

```diff
@@ -202,14 +202,13 @@ public async Task<AzureOperationResponse<IPage<BackupEngineBaseResource>>> ListW
             string _responseContent = null;
             if ((int)_statusCode != 200)
             {
-                var ex = new CloudException(string.Format("Operation returned an invalid status code '{0}'", _statusCode));
+                var ex = new NewErrorResponseException(string.Format("Operation returned an invalid status code '{0}'", _statusCode));
```

Comment:
This is a breaking change.  This will need to be reviewed by the breaking change board (please link the swagger, as this is likely the origin of the break)

Reply:
This is the Swagger PR: https://github.com/Azure/azure-rest-api-specs/pull/12611

There was no intentional breaking change in the swagger PR. There were only additive changes in Swagger to specify a default response to the existing APIs.

Reply:
@markcowl , Can you please check swagger PR mentioned above since this change is not a functional but it is mandatory requirement to have default error response returned by all APIs to swagger PR to go thorough.

Reply:
Hi @markcowl, I've updated swagger and regenerated the SDK. This change is no longer present in this PR.